### PR TITLE
unstyled search bar

### PR DESCRIPTION
**Pull request by tcjiang9**
manual testing

### CODE REVIEW

**Review by dmohs — COMMENTED** [review]
Fix your lint and then I'll take a look.

File: src/pages/library/DataBrowser.js [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
 const DataBrowser = () => {
+  const items = ['test1', 'test2']
+
+  const [searchFilter, setSearchFilter] = useState()
+
+  const filterBySearch = items => {
+    const lowerSearch = _.toLower(searchFilter)
+    return _.isEmpty(lowerSearch) ?
+      items :
+      _.filter(item => _.includes(lowerSearch, item), items)
```
Comment (dmohs) [inline src/pages/library/DataBrowser.js]:
```suggestion
    return _.isEmpty(lowerSearch) ? items : _.filter(item => _.includes(lowerSearch, item), items)
```
Optional. We value concision, so separating out these lines might not be worth doing. This is a very straightforward concept, so it should probably take up as little space as possible.

File: src/pages/library/DataBrowser.js [hunk 2]
```diff
@@ -1,11 +1,35 @@
...
+  }
+
+  const filteredItems = filterBySearch(items)
+
```
Comment (dmohs) [inline src/pages/library/DataBrowser.js]:
```suggestion
```

File: src/pages/library/DataBrowser.js [hunk 3]
```diff
@@ -1,11 +1,35 @@
...
-    libraryTopMatter('browse & explore')
+    libraryTopMatter('browse & explore'),
+    h(DelayedSearchInput, {
+      style: { flex: 1, marginLeft: '1rem' },
+      'aria-label': 'Search Featured Workspaces',
+      placeholder: 'Search Name or Description',
+      value: searchFilter,
+      onChange: setSearchFilter
+    }),
+    h(Fragment, filteredItems)
```
Comment (dmohs) [inline src/pages/library/DataBrowser.js]:
```suggestion
    h(Fragment, filterBySearch(items))
```
No reason to force the reader to look up how this is defined, better to show them.

**Review by dmohs — APPROVED** [review]
Approved with suggestions.

File: src/pages/library/DataBrowser.js (hunk 1 above)
Comment (dmohs) [reply src/pages/library/DataBrowser.js]: Actually, Utils.textMatch might be more appropriate here. Please take a look.

**Review by pshapiro4broad — APPROVED** [review]

**Pull request closed without merging**